specification/communication/data-plane/CallingServer/preview/2022-04-07-preview/communicationservicescallingserver.json

```diff
@@ -891,6 +932,62 @@
         "callbackUri": {
           "description": "The callback uri.",
           "type": "string"
+        },
+        "mediaStreamingConfiguration": {
+          "$ref": "#/definitions/MediaStreamingConfigurationDto",
+          "description": "Media Streaming Configuration."
+        }
+      }
+    },
+    "MediaStreamingTransportTypeDto": {
+      "description": "The type of tranport to be used for media streaming, eg. Websocket",
+      "enum": [
+        "websocket"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "MediaStreamingTransportTypeDto",
+        "modelAsString": true
+      }
+    },
+    "MediaStreamingContentTypeDto": {
+      "description": "Content type to stream, eg. audio, audio/video",
```

Comment:
Do we only support "audio" initially? If yes, then should probably update the description?